src/components/product-switcher/index.tsx

```diff
@@ -157,7 +157,7 @@ const ProductSwitcher: React.FC = () => {
             s.switcherOptionAnchor,
             s.switcherOptionContainer
           )}
-          href={product.url}
+          href={`/${product.slug}`}
```

Comment:
This change made the `url` property no longer needed on a product object. This was the only place the property was used.